### DISCUSSION TITLE
Decode Documentation for Github installers

### DISCUSSION THREAD

**Discussion by lostbits** [clangd/clangd#1162] [General]
I'm trying to download clangd in a Windows environment for Netbeans 13.0. Getting to the repository is easy but there is no decoder ring to determine which installation software to download. Is there any documentation source where this is available, and if not, would it be useful to put it into the [homepage](https://clangd.llvm.org/installation)?

What I guess is that the naming convention is:
clang+llvn-VERSION-HARDWARE-ENVIRONMENT-OS.tar.xz, as in, clang+llvm-14.0.0-amd64-pc-solaris2.11.tar.xz

What I don't see is which release is for Windows. Is it -i386, amd64, x86-64? And since for each of these there is an OS given and none says anything close to windows, which OS is appropriate.

**Comment by HighCommander4** [comment]
> What I guess is that the naming convention is: clang+llvn-VERSION-HARDWARE-ENVIRONMENT-OS.tar.xz, as in, clang+llvm-14.0.0-amd64-pc-solaris2.11.tar.xz

Where are you seeing these files?

If I go to https://github.com/clangd/clangd/releases, under the latest release, 14.0.3, under assets I see `clangd-windows-14.0.3.zip`. It doesn't contain an installer, but rather a portable binary package that you can unzip and use from anywhere.

**Comment by Yurii-huang** [comment]
The program supports window/linux. It is a program that compiles and builds closures. And there is no need to rely on the runtime of the library.
